PROJECT

```diff
@@ -78,10 +78,18 @@ resources:
   version: v1alpha1
 - api:
     crdVersion: v1
-    namespaced: false
```

Comment:
ThinRuntime should be cluster scope.

Reply:
You mean ThinRuntimeProfile?

Reply:
It is deleted by `kubebuilder create api`, not set `true` means `not namespaced scope`.

Reply:
Yes， `ThinRuntimeProfile`. Ok, I got it.